php/Dockerfile

```diff
@@ -71,6 +79,9 @@ RUN wget https://storage.googleapis.com/omnpackages/composerProxy-linux -O /usr/
 RUN curl -sL https://deb.nodesource.com/setup_10.x | bash - && \
     apt-get install -y nodejs
 
+# Install Yarn
+RUN apt-get install -y yarn
```

Comment:
This package seems to be already installed through `npm` in the following lines. By the way, it could be easier to read if you keep all these instructions together at the beginning of the file.

Reply:
I've removed the line and merge Yarn installation with the one for Grunt